src/sentry/utils/mockdata/core.py

```diff
@@ -1291,7 +1291,7 @@ def create_mock_user_feedback(project, has_attachment=True):
         "event_id": str(uuid.uuid4()).replace("-", ""),
         "timestamp": time.time(),
         "received": "2024-4-27T22:23:29.574000+00:00",
-        "environment": "prod",
```

Comment:
Got it! Seems like making an occurrence for a released group type will auto-create the environment if it doesn't exist. Not sure what it was doing before, since "prod" doesn't exist